code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -615,11 +615,11 @@
 /obj/machinery/nuclearbomb/beer/really_actually_explode()
 	disarm()
 
-/proc/KillEveryoneOnZLevel(z)
-	if(!z)
+/proc/KillEveryoneOnZCategory(zLevel)
+	if(!zLevel)
```

Comment:
it is not just zLevel. it is something greater concept than it.
also do not uae capitals at any of proc name and varname.